socorro/processor/mozilla_transform_rules.py

```diff
@@ -625,6 +625,9 @@ class BetaVersionRule(Rule):
     #: If we know it's good, cache it for 24 hours because it won't change
     LONG_CACHE_TTL = 60 * 60 * 24
 
+    #: Products that are on Buildhub--we don't want to ask Buildhub about others
+    BUILDHUB_PRODUCTS = ['firefox', 'fennec', 'fennecandroid']
```

Comment:
Putting "fennec" and "fennecandroid" in here because we're getting builds for two different build ids and crash reports for one of them have their product rewritten by the `ProductRewriteRule`. I want to get rid of that rule, but haven't done it, yet. In the meantime, seems prudent to handle both.